mq/publishers.go

```diff
@@ -13,6 +14,8 @@ import (
 	"golang.org/x/exp/slog"
 )
 
+var batch int
+
```

Comment:
why is this var declared here?

Reply:
> why is this var declared here?

For reading the env once, not reading the env every time when the func is called.

Reply:
var batch = servercfg.GetPeerUpdateBatchSize()

Reply:
we should also have a config to turn off batchupdates

Reply:
> var batch = servercfg.GetPeerUpdateBatchSize()

this is to call the env multiple times.  Anyway,  I will make change in next commit.